app/components/Product/product-details-description.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button } from "../ui/button";
+import { Separator } from "../ui/separator";
+
+export default function ProductDetailsDescription() {
+  return (
+    <div>
+      <p className="font-integralCF text-2xl leading-9 text-black  md:text-4xl">
+        One Life Graphic T-shirt
+      </p>
+      <div className="flex gap-3">
+        {/* <img src="/images/icon-rating.svg" alt="Rating Stars" /> */}
+        <img
+          src="/images/rating-stars-big.svg"
+          alt="Rating Stars"
```

Comment:
Jeżeli obrazek jest tylko podglądowy to powinien mieć pusty alt, jeśli jest istotny z perspektywy screen readerów to powinien mieć opis co ten obrazek reprezentuje czyli w tym wypadku "Ocena produktu 4.5/5"